recipes-bsp/bootfiles/rpi-config_git.bb

```diff
@@ -178,6 +178,13 @@ do_deploy() {
         echo "dtoverlay=${VC4DTBO}" >> ${DEPLOYDIR}/${BOOTFILES_DIR_NAME}/config.txt
     fi
 
+    # Choose Camera Sensor to be used, default imx219 sensor
+    if [ "${RASPBERRYPI_CAMERA_V2}" = "1" ]; then
+        echo "# Enable Sony RaspberryPi Camera" >> ${DEPLOYDIR}/${BOOTFILES_DIR_NAME}/config.txt
+        echo "dtoverlay=imx219" >> ${DEPLOYDIR}/${BOOTFILES_DIR_NAME}/config.txt
+        echo "core_freq_min=250" >> ${DEPLOYDIR}/${BOOTFILES_DIR_NAME}/config.txt
```

Comment:
Why do we need this?

Reply:
Since libcamera opensource is using the bcm2835-unicam device to access the camera, we need to explicitly set the device tree overlay in the config,txt.

Then only, libcamera could able to acquire and access the unicam device RaspberrypI cAMERA v2.

Please have a look into following libcamera installation docs for your reference.

https://github.com/raspberrypi/documentation/blob/master/linux/software/libcamera/README.md

Reply:
I was asking about `core_freq_min`. I see in the readme you pointed out that it is a workaround. Do we know if it was fixed? Can we can it out?

> Currently we also need to update the GPU's core_freq_min though this will become unnecessary in due course after further updates.

Reply:
Yes, in the latest driver, we no need this core_freq_min, and we had removed this now.

Reply:
Good stuff!